src/Paprika/Store/LeafOverflowPage.cs

```diff
@@ -15,15 +15,16 @@ namespace Paprika.Store;
 
     private ref Payload Data => ref Unsafe.AsRef<Payload>(page.Payload);
 
-    [StructLayout(LayoutKind.Explicit, Size = Size)]
+    [InlineArray(Size)]
+    [StructLayout(LayoutKind.Sequential, Pack = sizeof(byte), Size = Size)]
```

Comment:
Persistent

Reply:
Sequential is fine, just don't move fields 🙂